src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -617,7 +619,7 @@
     </PackageReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(NetFxSource)tools\targets\GenerateResourceStringsSource.targets" />
+  <Import Project="$(ToolsDir)targets\GenerateResourceStringsSource.targets" />
```

Comment:
Let's not change this as it's not related to this PR.
Also the new files are not required to be added, please remove them.

Reply:
done.